src/hooks.tsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 
+export const safeUseLayoutEffect = typeof window !== 'undefined' ? React.useLayoutEffect : () => {};
```

Comment:
neither `useEffect` or `useLayoutEffect` will run on the server, but `useLayoutEffect` will log a warning on the server, hence this